Description: don't use AM_INIT_AUTOMAKE macro when we aren't using automake
 Calling AM_INIT_AUTOMAKE() in configure.in serves no purpose if we're not
 using automake, and it confuses autoreconf.  Use AC_INIT() instead.
Author: Steve Langasek <vorlon@debian.org>

--- a/configure.in
+++ b/configure.in
@@ -26,7 +26,8 @@ dnl Configure.in for OpenLDAP
 AC_COPYRIGHT([[Copyright 1998-2014 The OpenLDAP Foundation. All rights reserved.
 Restrictions apply, see COPYRIGHT and LICENSE files.]])
 AC_REVISION([$Id: 81bd528fb5194c83d688db355737b7715448b958 $])
-AC_INIT([OpenLDAP],,[http://www.openldap.org/its/])
+AC_INIT([OpenLDAP],[$OL_VERSION],[http://www.openldap.org/its/])
+AC_PROG_MAKE_SET
 m4_define([AC_PACKAGE_BUGREPORT],[<http://www.openldap.org/its/>])
 AC_CONFIG_SRCDIR(build/version.sh)dnl
 dnl ----------------------------------------------------------------
@@ -69,7 +70,6 @@ dnl Determine host platform
 dnl		we try not to use this for much
 AC_CANONICAL_TARGET([])
 
-AM_INIT_AUTOMAKE([$OL_PACKAGE],[$OL_VERSION], [no defines])dnl
 AC_SUBST(PACKAGE)dnl
 AC_SUBST(VERSION)dnl
 AC_DEFINE_UNQUOTED(OPENLDAP_PACKAGE,"$PACKAGE",Package)
